src/transformers/models/mistral/modeling_mistral.py

```diff
@@ -314,11 +311,7 @@ def forward(
         key_states = key_states.view(bsz, q_len, self.num_key_value_heads, self.head_dim).transpose(1, 2)
         value_states = value_states.view(bsz, q_len, self.num_key_value_heads, self.head_dim).transpose(1, 2)
 
-        kv_seq_len = key_states.shape[-2]
-        if past_key_value is not None:
-            kv_seq_len += cache_position[0]
```

Comment:
Unwanted change I think

Reply:
Indeed not linked to the rotary though, but as I'm cleaning stuff, let's remove it as the same time